enterprise-ws-spec/src/main/asciidoc/Overview.adoc

```diff
@@ -152,18 +154,11 @@ at the possible expense of application portability.
 
 === Standard Services
 
-The Java EE platform defines a set of standard services a Java EE
-provider must supply. The Web Services for Java EE specification
+The Jakarta EE platform defines a set of standard services a Jakarta EE
+provider must supply. The Web Services for Jakarta EE specification
 identifies an additional set of run-time services that are required.
 
-==== JAX-RPC1.1
-
-JAX-RPC 1.1 provides run-time services for marshalling and demarshalling
-Java data and objects to and from XML SOAP messages. In addition,
-JAX-RPC defines the WSDL to Java mappings for a Service Endpoint
-Interface and a Service class.
-
-==== JAX-WS 2.2
+==== JAX-WS 3.0
```

Comment:
We should avoid using `JAX-WS` acronym if possible, now the spec itself is called `Jakarta XML Web Services`

Reply:
btw: I would probably avoid mentioning explicit version, since even though it is `3.0`, it is the first version under the new name, so perhaps say somewhere that `3.0` is assumed unless explicitly noted otherwise

Reply:
Hey @lukasj thanks for the quick feedback.  I think I've addressed everything so far.  I'll plan to continue updating JAX-WS -> Jakarta XML Web Services in future chapters, and avoid using versions unless the text specifically references one version.